Makefile.toml

```diff
@@ -72,7 +86,7 @@ command = "cargo"
 args = ["udeps", "--all-targets"]
```

Comment:
Maybe you missed this comment in the commit. Change this dependency with

```
[tasks.udeps_minimal]
[tasks.udeps-minimal]
dependencies = ["udeps-inst"]
toolchain = "nightly"
command = "cargo"
args = ["udeps", "--no-default-features"]

[tasks.udeps]
dependencies = ["udeps-inst", "udeps-minimal"]
toolchain = "nightly"
command = "cargo"
args = ["udeps", "--all-targets", "--all-features"]
```

Because we want to be sure that we don't have unused crate also when no features are enabled.